Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/cluster/ClusterDetail.java

```diff
@@ -319,4 +320,13 @@ private String getDefaultContainerName(String containerAddress){
 
         return null;
     }
+
+    public String getLivyConnectionUrl() {
```

Comment:
I would like to suggest to introduce a new interface `LivyCluster` with these methods.